packages/interactive_media_ads/ios/interactive_media_ads/Package.swift

```diff
@@ -15,11 +15,15 @@ let package = Package(
   products: [
     .library(name: "interactive-media-ads", targets: ["interactive_media_ads"])
   ],
-  dependencies: [],
+  dependencies: [
+    .package(
+      url: "https://github.com/googleads/swift-package-manager-google-interactive-media-ads-ios",
+      from: "3.23.0")
+  ],
   targets: [
     .target(
       name: "interactive_media_ads",
-      dependencies: [],
+      dependencies: ["GoogleInteractiveMediaAds"],
```

Comment:
I _think_ it's supposed to look like this:
```suggestion
      dependencies: [
        .product(name: "GoogleInteractiveMediaAds", package: "swift-package-manager-google-interactive-media-ads-ios")
      ],
```
Examples:
https://github.com/IABTechLab/uid2-ios-plugin-google-ima/blob/703d189bc9d5cf9ef7f2bc95c177aefbac6501b1/Package.swift#L26
https://github.com/bluebillywig/bbnativeplayerkit-cocoapod/blob/372309cf3b9bc0eea6ae828e89f68c0d78dfc974/Package.swift#L29
https://github.com/intergi/PlaywireMobile/blob/558ebfed737c2ae17ceb4ded619aedc421ef6ab5/Package.swift#L28

Reply:
This worked! Thanks!